src/Client.ts

```diff
@@ -731,7 +733,7 @@ export default class Client<ContentTypes = any> {
     for (const env of envelopes) {
       if (!env.message) continue
       try {
-        const res = await mapper(env)
+        const res = await mapper(env as EnvelopeWithMessage)
```

Comment:
updated the types here as the mapper never receives an envelope without a message